internal/server/v2/observation/golden/contained_in_all/FRA_AA2.json

```diff
@@ -5360,18 +5360,6 @@
         },
         "nuts/FR71": {
           "ordered_facets": [
-            {
-              "facet_id": "1456184638",
```

Comment:
Deletion of single point series from WikipediaStatsData import. Avaialble alternate source WikidataPopulation